src/urllib3/util/ssltransport.py

```diff
@@ -100,7 +100,10 @@ def recv(self, buflen: int = 1024, flags: int = 0) -> Union[int, bytes]:
         return self._wrap_ssl_read(buflen)
 
     def recv_into(
```

Comment:
Hmmm, Python doesn't allow passing `None` for it's `buffer` argument of `socket.recv_into()`? We probably shouldn't allow this either? Thoughts on this @jalopezsilva?